rs/registry/canister/src/flags.rs

```diff
@@ -11,8 +9,7 @@ pub(crate) fn is_chunkifying_large_values_enabled() -> bool {
     IS_CHUNKIFYING_LARGE_VALUES_ENABLED.get()
 }
 
-#[cfg(test)]
-pub(crate) fn temporarily_enable_chunkifying_large_values() -> Temporary {
+pub fn temporarily_enable_chunkifying_large_values() -> Temporary {
```

Comment:
This probably should remain a test-only method, and the static value could use a feature flag config.